examples/dataproc_provisioner_example/main.tf

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
nit: call the dir dataproc_provisioner. Adding example in the dir name is redundant.

p.s. while not really talking about dir names, I recommend reading https://blog.golang.org/package-names for good naming conventions to apply generally.

Reply:
Thanks for the tip!